src/rdf.rs

```diff
@@ -26,6 +26,12 @@ pub struct Graph {
     pub triples: Vec<Triple>,
 }
 
+/// Reference to a [Graph]
+#[derive(Debug, Clone, Default)]
+pub struct GraphRef<'a> {
```

Comment:
In this implementation, perhaps none. It could be changed though to be a more graph-like implementation.
`sophia` uses a trait with a lot of methods... https://docs.rs/sophia_api/0.7.0/sophia_api/graph/trait.Graph.html

Reply:
Traits addiction :zany_face: I'm out of my depth so I don't really have an opinion